committers.md

```diff
@@ -26,7 +26,7 @@ navigation:
 |Robert Evans|Oath|
 |Wenchen Fan|Databricks|
 |Joseph Gonzalez|UC Berkeley|
-|Thomas Graves|Oath|
+|Thomas Graves|Nvidia|
```

Comment:
Super nit but isn't the company name NVIDIA :)

Reply:
Yes it should be.. thanks :)